test/bootstrap.php

```diff
@@ -18,10 +18,7 @@
 define('PARATEST_ROOT', dirname(__DIR__));
 
 //check for .bat first if on windows.
```

Comment:
From what I can see, you don't check for the phpunit.bat file anymore on Windows, so this comment is outdated and should be removed.